app/services/common_platform/api/defendant_finder.rb

```diff
@@ -15,30 +14,50 @@ def call
     private
 
       def common_platform_defendant
-        @common_platform_defendant ||= common_platform_prosecution_case&.defendants&.find { |d| d.id.eql?(defendant_id) }
+        return @common_platform_defendant if @common_platform_defendant
+        return unless prosecution_case_urns
+
+        prosecution_case_urns.each do |urn|
+          prosecution_case = common_platform_prosecution_case(urn)
+
+          next unless prosecution_case
+
+          @common_platform_defendant ||= prosecution_case&.defendants&.find { |d| d.id.eql?(defendant_id) }
```

Comment:
The &. before .find is unnecessary because line 23

Reply:
Good spot! `prosecution_case` can't be nil but I suppose `defendants` still could be technically, so we can lose the first `&` but not the second one.